Components/ListOfCharts/index.js

```diff
@@ -3,14 +3,19 @@ import { FlexContainer } from "../FlexContainer";
 import ChartItem from "../ChartItem";
 import { chartArray } from "../../lib/listOfPlotTypes";
 
-export default function ListOfCharts() {
+export default function ListOfCharts({ onSelectChartType }) {
   return (
     <>
       <Heading>Step 2: Select the type of chart</Heading>
-      <FlexContainer $direction="row">
+      <FlexContainer $grid="grid">
```

Comment:
```suggestion
      <Container $grid="grid">
```

Flex and Grid are two completely different layout tools, either you name the styled component "Container" or you create a GridContainer and a FlexContainer as two separate things :)

Reply:
Done.